### DISCUSSION TITLE
fusermount: mounting over filesystem type 0x858458f6 is forbidden | debian testing

### DISCUSSION THREAD

**Discussion by staltux** [AppImage/AppImageKit#1338] [Q&A] [answered]
using libfuse (2.9.9-8.1) on debian testing

**Comment by probonopd** [comment]
Hi @staltux, what is filesystem type `0x858458f6`?

What does `mount | grep tmp` say?

Does the situation improve if you `sudo mount -t tmpfs tmpfs /tmp`?

AppImages get mounted within `/tmp`, and `/tmp` used to be `tmpfs`, which is whitelisted by FUSE in
https://github.com/libfuse/libfuse/blob/f88e08f34d2d4f398f23797707e1c50cd306e405/util/fusermount.c#L1137C3-L1137C19

Reference:
* https://github.com/libfuse/libfuse/issues/982

**Reply by staltux** [reply, marked as answer]
0x858458f6 is ramfs
and yes, mounting tmp has tmpfs works, thanks
edit:
is not ideal because tmpfs don't have limit on ram usage, but is a temporary tradeoff

**Reply by probonopd** [reply]
Is Debian using ramfs by default or did you specifically set it up this way?

**Reply by staltux** [reply]
Non default, the normal way is just use /tmp as regular folder and the system clean up it on reboot
